repo/packages/K/kafka/1/marathon.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "{{kafka.app.id}}",
```

Comment:
can we make this kafka.framework-name ?   the issue is if I install 2 versions of kafka, I am force to change to variables for this to work.  The framework-name and the app.id.